frontend/benefit/handler/public/locales/fi/common.json

```diff
@@ -1044,4 +1047,4 @@
     },
     "keyword": "Hakusana"
   }
-}
```

Comment:
I think that the editors should be configured so that they don't mess up the line endings.

Reply:
This is green now, not worth waiting for builds. Maybe merge and do a hotfix to address the line change issues?